src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems

```diff
@@ -653,7 +653,6 @@
     <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\InteropServices\CoClassAttribute.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\InteropServices\CollectionsMarshal.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\InteropServices\ComDefaultInterfaceAttribute.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\InteropServices\ComEventInterfaceAttribute.cs" />
```

Comment:
It does not matter a whole lot where this lives anymore. 

We have been generally moving things into CoreLib (see https://github.com/dotnet/runtime/issues/2138), and the legacy COM interop is too tightly coupled with the runtime that good chunk of it is always going to live in CoreLib, so we are not going to be 100% clean.

If you prefer to have as many COM interop things out of CoreLib, I am fine with this change. You can probably other things too (e.g. `ComMemberType`).

Reply:
BTW: As I have been looking through this, I have noticed that GetMethodInfoForComSlot and GetComSlotForMethodInfo in VM are dead code. You can delete those.

Reply:
> If you prefer to have as many COM interop things out of CoreLib

If you are doing this just to fix #22174 and don't have opinion, then it may better to just close #22174 as no longer relevant.

Reply:
I really don't actually. However, I can do some bookkeeping here.